scripts/imm_tests.bats

```diff
@@ -0,0 +1,182 @@
+export LABEL_LIST=" airliner, Arctic fox, bee, golden retriever, gorilla, magnetic compass, peacock, pelican, snail, zebra"
+export JPEGS_INFERENCE_ACCURACY=0
+export SERVER_CERT="server-tf.crt"
+export CLIENT_CERT="client-tf.crt"
+export CLIENT_KEY="client-tf.key"
+export SAVED_MODEL_SRC="https://storage.googleapis.com/inference-eu/models_zoo/resnet_V1_50/saved_model/saved_model.pb"
+export IMAGES_NUMPY_SRC="https://storage.googleapis.com/inference-eu/models_zoo/resnet_V1_50/datasets/10_v1_imgs.npy"
+
+[[ ! -f ${MODEL_PATH} ]] && echo "Downloading model" && wget ${SAVED_MODEL_SRC}
+[[ ! -f ${NUMPY_PATH} ]] && echo "Downloading numpy images" && wget ${IMAGES_NUMPY_SRC}
+
+. ./imm_utils.sh
+
+get_inference_accuracy(){
+    NUMBER_OF_MATCHES=0
+    IFS=',' read -ra LABELS <<< "$LABEL_LIST"
+    for i in "${LABELS[@]}"; do
+         [[ $* =~ "$i" ]] && let "NUMBER_OF_MATCHES++"
+    done
+    JPEGS_INFERENCE_ACCURACY=$((NUMBER_OF_MATCHES*10))
+    echo "${JPEGS_INFERENCE_ACCURACY}"
+}
+
+
+[[ ! -f ${MODEL_PATH} ]] && echo "Downloading model" && wget https://storage.googleapis.com/inference-eu/models_zoo/resnet_V1_50/saved_model/saved_model.pb
+[[ ! -f ${NUMPY_PATH} ]] && echo "Downloading numpy images" && wget https://storage.googleapis.com/inference-eu/models_zoo/resnet_V1_50/datasets/10_v1_imgs.npy
+
+#echo "****************************ADMIN****************************"
+@test "Get admin token" {
+    response="$(get_token admin)"
+    grep -E "Token is saved" <<< $response
+}
+
+@test "Create tenant" {
+    response="$(yes | ./imm c t ${TENANT_NAME} ${ADMIN_SCOPE})"
+    grep -E "${TENANT_NAME}" <<< $response
+}
+
+@test "List tenants" {
+    response="$(./imm ls t)"
+    grep -E "${TENANT_NAME}" <<< $response
+}
+
+@test "Logout" {
+    response="$(yes | ./imm logout)"
+    grep -E "Signed out" <<< $response
+}
+
+#echo "*****************************USER*****************************"
+@test "Get user token" {
+    response="$(get_token user)"
+    grep -E "Token is saved" <<< $response
+}
+
+
+@test "Model upload" {
+    response="$(./imm u ${MODEL_PATH} ${MODEL_NAME} ${MODEL_VERSION} ${TENANT_NAME})"
+    echo "$response" >&3
+    grep -E "completed successfully" <<< $response
+}
+
+@test "List models" {
+    response="$(./imm ls m ${TENANT_NAME})"
+    grep -E "${MODEL_NAME}" <<< $response
+}
+
+@test "Create endpoint" {
+    response=`./imm c e ${ENDPOINT_NAME} ${MODEL_NAME} "${MODEL_VERSION_POLICY}" ${TENANT_NAME} ${SERVING_NAME}`
+    echo "$response" >&3
+    grep -E "${ENDPOINT_NAME}-${TENANT_NAME}.${DOMAIN_NAME}" <<< $response
+}
+
+@test "List endpoints" {
+    response="$(./imm ls e ${TENANT_NAME})"
+    grep -E "${ENDPOINT_NAME}" <<< $response
+}
+
+@test "Check if endpoint is up and running" {
+    sleep 20
```

Comment:
Could we make it smaller?

Reply:
For me 20s is a "save option" (i tested it on dev machine)
In cloud it could be smaller, but i decided to keep it that way.